### PR TITLE
Obtain WASM/JS files from emscripten-forge

### PR DESCRIPTION
**Pull request by ianthomas23**
This PR removes the WASM/JS files from this repo and instead obtains them at build time from Emscripten-forge, based on the work in #25. Co-authored by @DerThorsten.

This passes the tests locally for me and works with `cockle-playground`. It currently doesn't work with the JupyterLite `terminal` extension but I think that is an existing issue in JupyterLite related to the recent upgrades and I will deal with that separately.

### CODE REVIEW

**Comment by ianthomas23** [comment]
Looks promising. Probably the package release process is broken now, but I'll deal with that at the next release.

File: package.json [hunk 1]
```diff
@@ -19,17 +19,20 @@
...
+        "fetch:wasm:create-env": "micromamba create -p $(pwd)/cockle_wasm_env -y cockle_fs grep coreutils --platform=emscripten-wasm32 -c https://repo.mamba.pm/emscripten-forge -c https://repo.mamba.pm/conda-forge",
+        "fetch:wasm:copy": "mkdir -p src/wasm && cp -r $(pwd)/cockle_wasm_env/bin/*.js src/wasm",
+        "fetch:wasm": "npm run fetch:wasm:create-env && npm run fetch:wasm:copy",
         "build": "tsc",
         "eslint": "npm run eslint:check -- --fix",
         "eslint:check": "eslint . --cache --ext .ts,.tsx",
         "lint": "npm run prettier && npm run eslint",
         "lint:check": "npm run prettier:check && npm run eslint:check",
-        "prepack": "npm install && npm run build",
+        "prepack": "npm install && npm run fetch:wasm && npm run build",
```
Comment (jtpio) [inline package.json:30]: > Looks promising. Probably the package release process is broken now, but I'll deal with that at the next release.

Isn't this change enough to pull the files at release time?

Comment (jtpio) [reply package.json:30]: Maybe we can check the files are properly included in the release artifacts built by the `check_release` workflow: https://github.com/jupyterlite/cockle/actions/runs/10300533031

Comment (ianthomas23) [reply package.json:30]: The change is enough for me locally so it should it should be fine for a release provided `micromamba` is set up properly. It looks I started but didn't finish that.

Comment (jtpio) [reply package.json:30]: Right, I see the `prep-release.yml` workflow was also updated to install `micromamba`, so it should hopefully work.

**Pull request merged**